GBPing/GBPing.m

```diff
@@ -388,7 +388,7 @@ -(void)listenOnce {
             NSMutableData *packet;
 
             packet = [NSMutableData dataWithBytes:buffer length:(NSUInteger) bytesRead];
-            assert(packet);
+            if (packet == nil) { return; }
```

Comment:
similar here, do I set `pingSummary.status` to `GBPingStatusFail` ?

Reply:
actually... I need the packet to create `pingSummary`.  ill probably just return here as this should be caught above I think anyway